apps/docs/components/Sidebar/sidebar.css

```diff
@@ -0,0 +1,132 @@
+.hd-sidebar {
+    border-inline-end: 1px solid var(--hd-color-border-primary-weak);
+    position: sticky;
+    top: var(--hd-space-8);
+    height: calc(100vh - var(--hd-space-8));
+    min-width: 18rem;
+    max-width: 18rem;
+}
+
+.hd-sidebar__container {
+    padding-inline: var(--hd-space-3);
+    overflow-y: scroll;
+    scroll-margin-top: var(--hd-space-8);
+    overscroll-behavior: contain;
+    height: calc(100% - var(--hd-space-1));
```

Comment:
I do not understand the purpose of this statement and the following one

Reply:
`overscroll-behavior: contain;` will prevent a bounce effect when the sidebar is scrolled to the bottom as we're not on the end of document and we don't want a bounce effect when at the bottom of this element.

This is inspired by Polaris Sidebar.

As for the height calc it is to prevent the scrollbar to be at the very top and bottom of the parent. Since it's at top --hd-space--05 the height should be 100% - --hd-space--05 X 2